docs_guide/sphinx_guide/how_to_build_docs.rst

```diff
@@ -0,0 +1,103 @@
+##################################
+How to build documentation locally
+##################################
+
+.. include:: ../others/how-to_type.rst
+
+This guide shows how to build the documentation of your project in a local machine.
+
+Pre-requisites
+==============
+
+This guide assumes your project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+:doc:`sphinx-quickstart <sphinx:man/sphinx-quickstart>`.
+
+Install package and dependencies
+================================
+
+Before trying to build your documentation, make sure you have installed:
+
+* The Qiskit package you want to build, that in this guide will be called ``qiskit_x``.
```

Comment:
Similarly to the "Qiskit project" term, I'm not sure whether we should keep using the "Qiskit package" term.

Reply:
Simplified to just "project"